### DISCUSSION TITLE
Migration scraping and date to another server

### DISCUSSION THREAD

**Discussion by Mojito2110** [prometheus/prometheus#12538] [Q&A]
Hello,

is it possible to implement the migration scenario below :
move the TSDB from a Prometheus 2.2x or 2.3x server to a Prometheus 2.4x server

Despite the version difference, is the move possible?
What are the steps involved in switching from one server to another while limiting data loss? I'm having trouble finding the details of the operations to be carried out for this type of scenario (stopping the servers, copying, etc.... ).

Thanks for your help

**Comment by Charlotte-br560** [comment]
Yes, it's possible to migrate TSDB from Prometheus 2.2x or 2.3x to a 2.4x server. Here are general steps:

Backup: Export data from the source Prometheus server.
Stop Servers: Stop both source and destination Prometheus servers.
Copy Data: Transfer the exported data to the destination server.
Import Data: Import the data into the destination Prometheus server.
Configuration Check: Ensure configurations match between servers.
Start Servers: Start both servers and verify data integrity.
For detailed instructions, refer to Prometheus documentation or community resources.

**Reply by truongnht** [reply]
@Charlotte-br560 instead of the above suggestion, can I use `remote_read` to migrate prometheus data?
